view/src/components/foodbanks.js

```diff
@@ -981,13 +950,13 @@ class Foodbank extends Component {
                   <Typography className={classes.pos} color="textSecondary">
                     Point of Contact:
                   </Typography>
-                  <Typography variant="body2" component="p">
-                    Name: {this.state.contactName}
+                  {/* <Typography variant="body2" component="p">
+                    Name: {this.state.contacts[0]["contactName"]}
                     <br />
-                    Phone: {this.state.contactPhone}
+                    Phone: {this.state.contacts[0]["contactName"]}
                     <br />
-                    Email: {this.state.contactEmail}
-                  </Typography>
+                    Email: {this.state.contacts[0]["contactEmail"]}
+                  </Typography> */}
```

Comment:
remove - you can look back thru revision history if you really need this code again